code/graphics/opengl/gropenglstate.cpp

```diff
@@ -334,7 +332,7 @@ GLboolean opengl_state::ClipPlane(GLint num, GLint state)
 
 GLboolean opengl_state::ClipDistance(GLint num, GLint state)
 {
-	Assert( (num >= 0) || (num < (int)(sizeof(clipdistance_Status) / sizeof(GLboolean))) || GL_version >= 30 );
+	Assert( (num >= 0) && (num < (int)(sizeof(clipdistance_Status) / sizeof(GLboolean))) );
```

Comment:
Ah, yeah; I just saw the structural change from "A or B or C" to "A and B" without actually looking at what it was checking. The old check was obviously wrong.